ui/src/app/applications/components/applications-list/applications-list.tsx

```diff
@@ -287,6 +289,15 @@ export const ApplicationsList = (props: RouteComponentProps<{}>) => {
     const clusters = React.useMemo(() => services.clusters.list(), []);
     const [isAppCreatePending, setAppCreatePending] = React.useState(false);
     const loaderRef = React.useRef<DataLoader>();
+    const [pageView, setPageTitle] = React.useState(AppsListViewKey.Tiles.toString());
+    const {List, Summary, Tiles} = AppsListViewKey;
+    React.useEffect(() => {
+        if (query.has('view')) {
```

Comment:
The view type should be taken from query or view preferences. The `<ViewPref>` performs that logic. So instead we can wrap the page in `<ViewPref>`, remove `const [pageView, setPageTitle] = React.useState(AppsListViewKey.Tiles.toString());` and just use `getPageTitle` to get calculate page title.

Reply:
Hmm, this is much better.

Reply:
ViewPref moved out to wrap page. Done.